assets/src/edit-story/style.css

```diff
@@ -21,7 +21,8 @@
  */
 
 #screen-meta,
-#screen-meta-links {
+#screen-meta-links,
+#screen-meta-links ~ .notice {
```

Comment:
I currently have an `.update_nag` on my screen:

![Screen Shot 2020-04-02 at 11 59 04 AM](https://user-images.githubusercontent.com/637548/78270866-69349380-74d9-11ea-96a4-a3ad4c07b5a3.png)

That wouldn't be caught by this, would it? And I feel it shouldn't go here - at least not in the editor!

HTML is:

```html
<div id="screen-meta-links">
  <div id="screen-options-link-wrap" class="hide-if-no-js screen-meta-toggle">
    <button type="button" id="show-settings-link" class="button show-settings" aria-controls="screen-options-wrap" aria-expanded="false">Screen Options</button>
  </div>
</div>
<div class="update-nag">
  <a href="https://wordpress.org/support/wordpress-version/version-5-4/">WordPress 5.4</a> is available! 
  <a href="http://one.wordpress.test/wp-admin/update-core.php" aria-label="Please update WordPress now">Please update now</a>.
</div>
```

Reply:
Good catch! That should definitely be hidden as well.

Reply:
(and you should definitely update 😄 )